### PR TITLE
Documentation updates Round 2!

### PR DESCRIPTION
**Pull request by AlannaBurke**
Documentation updates! This includes typo cleanup, language simplification, and more!

If you have access to readthedocs, I also built a private version for review here: https://lagoon.readthedocs.io/en/documentation-updates

### CODE REVIEW

File: docs/using_lagoon/drupal/subfolders.md [hunk 1]
```diff
@@ -1,68 +1,107 @@
...
+
+```text
 COPY location_prepend.conf /etc/nginx/conf.d/drupal/location_prepend.conf
 RUN fix-permissions /etc/nginx/conf.d/drupal/*
 ```
 
 ## Modifications of subfolder application
 
-Like the root application, we also need to teach the subfolder application that it is running under a subfolder, for this we create two files:
+<<<<<<< HEAD
```
Comment (smlx) [inline docs/using_lagoon/drupal/subfolders.md]: Looks like a couple of merge/rebase artefacts here..

File: docs/using_lagoon/drupal/subfolders.md [hunk 2]
```diff
@@ -1,68 +1,107 @@
...
+Like the root application, we also need to teach the subfolder application \(in this example, the Drupal installation for www.example.com/blog\), that it is running under a subfolder. To do this, we create two files:
+>>>>>>> master
 
-#### `location_drupal_append_subfolder.conf`
+### `location_drupal_append_subfolder.conf`
 
-create file `location_drupal_append_subfolder.conf`:
+Create a file called `location_drupal_append_subfolder.conf` in the root of your subfolder Drupal installation:
+
+<<<<<<< HEAD
```
Comment (smlx) [inline docs/using_lagoon/drupal/subfolders.md]: .. and here.

File: docs/administering_lagoon/install.md [hunk 3]
```diff
@@ -1,80 +1,84 @@
...
-Once Lagoon is install operational you need to initialise OpendistroSecurity to allow Kibana multitenancy. This only needs to be run once in a new setup of lagoon.
+1. Open a shell of an Elasticsearch pod in `logs-db`.
+2. `run ./securityadmin_demo.sh`.
 
-1. Open a shell of an elasticsearch pod in logs-db.
-2. run ./securityadmin_demo.sh
+## Configure Installed Lagoon
 
-### Configure Installed Lagoon
+We have a fully running Lagoon. Now it's time to configure the first project inside of it. Follow the examples in [GraphQL API](https://github.com/AlannaBurke/lagoon/tree/b9add2d0bfd8db43295d7ccdab7729436674f764/administering_lagoon/graphql_api.md)
```
Comment (tobybellwood) [inline docs/administering_lagoon/install.md]: (selflink here)

File: docs/using_lagoon/index.md [hunk 4]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
+| Type | Versions | Dockerfile |
+| :--- | :--- | :--- |
+| [Elasticsearch](docker_images/elasticsearch.md) | 6.8.2, 7.1.1, 7.3.0 | [elasticsearch/Dockerfiles](https://github.com/amazeeio/lagoon/tree/master/images/elasticsearch) |
+| [MariadB-Drupal](docker_images/mariadb-drupal.md) | 10 | [mariadb-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mariadb-drupal/Dockerfile) |
+| [MariaDB](docker_images/mariadb.md) | 10 | [mariadb/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mariadb/Dockerfile) |
+| [MongoDB](docker_images/mongo.md) | 3.6 | [mongo/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mongo/Dockerfile) |
+| [Nginx](docker_images/nginx.md) | openresty/1.15.8.2 | [nginx/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/nginx/Dockerfile) |
+| [Nginx-Drupal](docker_images/nginx-drupal.md) |  | [nginx-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/nginx-drupal/Dockerfile) |
+| [php-fpm-Drupal](docker_images/php-cli-drupal.md) | 7.2, 7.3 | [php/cli-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/php/cli-drupal/Dockerfile) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [php-fpm-Drupal](docker_images/php-cli-drupal.md) | 7.2, 7.3, 7.4 | [php/cli-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/php/cli-drupal/Dockerfile) |
```

File: docs/using_lagoon/index.md [hunk 5]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
+| [php-cli](docker_images/php-cli/) | 7.2, 7.3 | [php/cli/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/php/cli/Dockerfile) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [php-cli](docker_images/php-cli/) | 7.2, 7.3, 7.4 | [php/cli/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/php/cli/Dockerfile) |
```

File: docs/using_lagoon/index.md [hunk 6]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
-| [solr](docker_images/solr.md) | 5.5, 6.6, 7.5 | [solr/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/solr/Dockerfile) |
-| [varnish-drupal](docker_images/varnish-drupal.md) | 5 | [varnish-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/varnish-drupal/Dockerfile) |
-| [varnish](docker_images/varnish.md) | 5 | [varnish/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/varnish/Dockerfile) |
-
-All images are pushed to https://hub.docker.com/u/amazeeio.
-We suggest always using the latest tag (like `amazeeio/nginx:latest`) or unsuffixed images (like `amazeeio/node:10`), as they are kept up to date in terms of features and security. If you choose to use a specific Lagoon version of an image like `amazeeio/nginx:v0.21.0` or `amazeeio/node:10-v0.21.0` it is your own responsibility to upgrade the version of the images as soon as a new Lagoon version is released!
+| [Postgres](docker_images/postgres.md) | 11.x | [postgres/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/postgres/Dockerfile) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [Postgres](docker_images/postgres.md) | 11.6 | [postgres/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/postgres/Dockerfile) |
```

File: docs/using_lagoon/index.md [hunk 7]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
+| [RabbitMQ](docker_images/rabbitmq.md) |  |  |
+| [Redis-persistent](docker_images/redis-permanent.md) | 5.0.0 | [redis-persistent/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/redis-persistent/Dockerfile) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [Redis-persistent](docker_images/redis-permanent.md) | 5.0 | [redis-persistent/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/redis-persistent/Dockerfile) |
```

File: docs/using_lagoon/index.md [hunk 8]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
+| [Redis](docker_images/redis.md) | 5.0.0 | [redis/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/redis/Dockerfile) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [Redis](docker_images/redis.md) | 5.0 | [redis/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/redis/Dockerfile) |
```

File: docs/using_lagoon/index.md [hunk 9]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
+| [Solr-Drupal](docker_images/solr-drupal.md) | 5.5, 6.6, 7.5 | [solr-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/solr-drupal/Dockerfile) |
+| [Solr](docker_images/solr.md) | 5.5, 6.6, 7.5 | [solr/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/solr/Dockerfile) |
+| [Varnish-Drupal](docker_images/varnish-drupal.md) | 5 | [varnish-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/varnish-drupal/Dockerfile) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [Varnish-Drupal](docker_images/varnish-drupal.md) | 5.2 | [varnish-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/varnish-drupal/Dockerfile) |
```

File: docs/using_lagoon/index.md [hunk 10]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
+| [Varnish](docker_images/varnish.md) | 5 | [varnish/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/varnish/Dockerfile) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [Varnish](docker_images/varnish.md) | 5.2 | [varnish/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/varnish/Dockerfile) |
```

File: docs/using_lagoon/index.md [hunk 11]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
-| [solr](docker_images/solr.md) | 5.5, 6.6, 7.5 | [solr/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/solr/Dockerfile) |
-| [varnish-drupal](docker_images/varnish-drupal.md) | 5 | [varnish-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/varnish-drupal/Dockerfile) |
-| [varnish](docker_images/varnish.md) | 5 | [varnish/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/varnish/Dockerfile) |
-
-All images are pushed to https://hub.docker.com/u/amazeeio.
-We suggest always using the latest tag (like `amazeeio/nginx:latest`) or unsuffixed images (like `amazeeio/node:10`), as they are kept up to date in terms of features and security. If you choose to use a specific Lagoon version of an image like `amazeeio/nginx:v0.21.0` or `amazeeio/node:10-v0.21.0` it is your own responsibility to upgrade the version of the images as soon as a new Lagoon version is released!
+| [Postgres](docker_images/postgres.md) | 11.x | [postgres/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/postgres/Dockerfile) |
+| [RabbitMQ](docker_images/rabbitmq.md) |  |  |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [RabbitMQ](docker_images/rabbitmq.md) | 3.8  |  |
```

File: docs/using_lagoon/index.md [hunk 12]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
-| [mariadb](docker_images/mariadb.md) | 10 | [mariadb/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mariadb/Dockerfile) |
-| [mongo](docker_images/mongo.md) | 3.6 | [mongo/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mongo/Dockerfile) |
-| [nginx](docker_images/nginx-drupal.md) | openresty/1.15.8.2 | [nginx/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/nginx/Dockerfile) |
-| [nginx-drupal](docker_images/nginx.md) | | [nginx-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/nginx-drupal/Dockerfile) |
-| [php-cli-drupal](docker_images/php-cli-drupal.md) | 7.2, 7.3 | [php/cli-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/php/cli-drupal/Dockerfile) |
-| [php-cli](docker_images/php-cli.md) | 7.2, 7.3 | [php/cli/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/php/cli/Dockerfile) |
+| Type | Versions | Dockerfile |
+| :--- | :--- | :--- |
+| [Elasticsearch](docker_images/elasticsearch.md) | 6.8.2, 7.1.1, 7.3.0 | [elasticsearch/Dockerfiles](https://github.com/amazeeio/lagoon/tree/master/images/elasticsearch) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [Elasticsearch](docker_images/elasticsearch.md) | 6.8, 7.1, 7.3 | [elasticsearch/Dockerfiles](https://github.com/amazeeio/lagoon/tree/master/images/elasticsearch) |
```

File: docs/using_lagoon/index.md [hunk 13]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
+| [MariadB-Drupal](docker_images/mariadb-drupal.md) | 10 | [mariadb-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mariadb-drupal/Dockerfile) |
+| [MariaDB](docker_images/mariadb.md) | 10 | [mariadb/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mariadb/Dockerfile) |
+| [MongoDB](docker_images/mongo.md) | 3.6 | [mongo/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mongo/Dockerfile) |
+| [Nginx](docker_images/nginx.md) | openresty/1.15.8.2 | [nginx/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/nginx/Dockerfile) |
+| [Nginx-Drupal](docker_images/nginx-drupal.md) |  | [nginx-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/nginx-drupal/Dockerfile) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [Nginx-Drupal](docker_images/nginx-drupal.md) | openresty/1.15.8.2 | [nginx-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/nginx-drupal/Dockerfile) |
```

File: docs/using_lagoon/index.md [hunk 14]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
-| [mongo](docker_images/mongo.md) | 3.6 | [mongo/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mongo/Dockerfile) |
-| [nginx](docker_images/nginx-drupal.md) | openresty/1.15.8.2 | [nginx/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/nginx/Dockerfile) |
-| [nginx-drupal](docker_images/nginx.md) | | [nginx-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/nginx-drupal/Dockerfile) |
-| [php-cli-drupal](docker_images/php-cli-drupal.md) | 7.2, 7.3 | [php/cli-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/php/cli-drupal/Dockerfile) |
-| [php-cli](docker_images/php-cli.md) | 7.2, 7.3 | [php/cli/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/php/cli/Dockerfile) |
+| Type | Versions | Dockerfile |
+| :--- | :--- | :--- |
+| [Elasticsearch](docker_images/elasticsearch.md) | 6.8.2, 7.1.1, 7.3.0 | [elasticsearch/Dockerfiles](https://github.com/amazeeio/lagoon/tree/master/images/elasticsearch) |
+| [MariadB-Drupal](docker_images/mariadb-drupal.md) | 10 | [mariadb-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mariadb-drupal/Dockerfile) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [MariadB-Drupal](docker_images/mariadb-drupal.md) | 10.2 | [mariadb-drupal/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mariadb-drupal/Dockerfile) |
```

File: docs/using_lagoon/index.md [hunk 15]
```diff
@@ -10,60 +10,61 @@ To run a Lagoon Project, your system must meet the requirements to run Docker. W
...
+| [MariaDB](docker_images/mariadb.md) | 10 | [mariadb/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mariadb/Dockerfile) |
```
Comment (tobybellwood) [inline docs/using_lagoon/index.md]:
```suggestion
| [MariaDB](docker_images/mariadb.md) | 10.2 | [mariadb/Dockerfile](https://github.com/amazeeio/lagoon/blob/master/images/mariadb/Dockerfile) |
```

**Comment by tobybellwood** [comment]
sorry @AlannaBurke - a heap of version nitpicks in as suggestions - a couple of updates, but mostly semver consistency - let's list out the minor versions, instead of patch (removing some maintenance overhead)

**Comment by AlannaBurke** [comment]
@tobybellwood no apologies needed! thanks for the details, that's stuff i'm not going to know!

**Comment by AlannaBurke** [comment]
Ok, I've reviewed every file, and checked every page both locally and on readthedocs. If there's any errors to be found, I'm not going to find them now! Bless anyone who reviews this enormous thing, try the rich text diff.

**Review by tobybellwood — APPROVED** [review]
more great work

**Pull request merged**